chcemvediet/apps/anonymization/models.py

```diff
@@ -27,10 +27,10 @@ class AttachmentNormalization(FormatMixin, models.Model):
                 True if normalization has succeeded, False otherwise.
                 """))
 
-    # May be NULL; Random local filename is generated in save() when creating a new object.
-    file = models.FileField(upload_to=u'attachment_normalizations', max_length=255, null=True,
+    # Filename may be empty; Random local filename is generated in save() when creating a new object.
+    file = models.FileField(upload_to=u'attachment_normalizations', max_length=255, blank=True,
             help_text=squeeze(u"""
-                NULL if normalization failed or normalization didn't create any file.
+                Empty filename if normalization failed or normalization didn't create any file.
```

Comment:
Keby si si to pretestoval a pozrel, co sa uklada do DB, zistil by si, ze `FileField` nepodporuje NULL hodnoty. Chybajuci subor reprezentuje prazdnym stringom.